Firestore/core/src/firebase/firestore/model/types.h

```diff
@@ -19,6 +19,10 @@
 
 #include <cstdint>
 
+#if defined(__OBJC__)
+#import <Foundation/Foundation.h>
```

Comment:
Forgot to mention: without this addition, I remember getting some errors when I tried to include this file from `stream_objc_bridge.h`), something about not being able to find basic Objective-C library types like `NSString`.